common-utils/README.md

```diff
@@ -174,6 +231,71 @@ f"{project_name}-prod-live"
 
 `prod` will be used for flows pushed to the `main-v2` branch.
 
+The purpose of this `deploy-flows` is to:
+
+- Using the `tool.prefect.flows_folder` configuration, find Python files that end in `_flow.py` and attempt to deploy them to Prefect.
+- The deployment of a flow consists of:
+    - Creating an ECS Task Block in Prefect that defines the flow's task definition.
+    - Loading the flow files to the S3 filessystem.
+    - Registering the flow's deployment configurations with Prefect.  A flow can have multiple deployment configurations.  Deployment documentation is found [here](https://docs.prefect.io/concepts/deployments/).
+
+#### **Flow Deployment and Deployment Configurations**
+
+The folder to look for flows is defined in the pyproject.toml.
+
+```toml
+[tool.prefect]
+flows_folder = "tests/test_flows"
```

Comment:
Will `test_flows` be where we put all production flows?

Reply:
no its just for the common-utils project...putting it here made it easier for unit tests.  When I create the data-products project they will be at `src/` most likely.